### PR TITLE
patches: Add 0169

### PR DESCRIPTION
**Pull request by feiwan1**
To support sliding window bitrate control for ffmpeg-vpl.

### CODE REVIEW

File: patches/0169-lavc-qsvenc-Add-sliding-window-bitrate-control-for-C.patch [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+@@ -1258,6 +1269,21 @@ static int init_video_param(AVCodecContext *avctx, QSVEncContext *q)
+                 q->extco3.TransformSkip = MFX_CODINGOPTION_UNKNOWN;
+             q->extco3.GPB              = q->gpb ? MFX_CODINGOPTION_ON : MFX_CODINGOPTION_OFF;
+         }
++
++#if QSV_HAVE_SW
++        if ((avctx->codec_id == AV_CODEC_ID_H264 || avctx->codec_id == AV_CODEC_ID_HEVC ||
++            avctx->codec_id == AV_CODEC_ID_AV1) && q->sw_size) {
++            //if (QSV_RUNTIME_VERSION_ATLEAST(q->ver, 2, 13)) {
++            if (1) {
```
Comment (xhaihao) [inline patches/0169-lavc-qsvenc-Add-sliding-window-bitrate-control-for-C.patch:77]: Please check the condition.

**Comment by wenqingx** [comment]
check the failed cases (4 cases) , it maybe a regression on windows, i will bisect it .

**Review by xhaihao — APPROVED** [review]
LGTM

**Comment by wenqingx** [comment]
> wenqingx

https://jira.devtools.intel.com/browse/VIZ-20932

**Pull request merged**